content/components/pages/OverviewPage.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import {
+  useEnvironmentState,
+  useSelectedNormandyEnvironmentAPI,
+} from "devtools/contexts/environment";
+import { PendingReviews } from "../overview/PendingReviews";
+import { Loader } from "rsuite";
+
+export const OverviewPage: React.FC<any> = ({}) => {
```

Comment:
In general, we shouldn't be using `any` in Typescript. It turns off most of the type safety. In this case since you don't take any props, you can just use the default value for the `FC` type parameter.

```suggestion
export const OverviewPage: React.FC = () => {
```